modules/smtp/modules.php

```diff
@@ -2157,15 +2166,20 @@ function default_smtp_server($user_config, $session, $request, $config, $user, $
  * @subpackage smtp/functions
  */
 if (!hm_exists('recip_count_check')) {
-function recip_count_check($headers, $omod) {
+function recip_count_check($headers, $omod, $reply_type='') {
     $headers = lc_headers($headers);
     $recip_count = 0;
-    if (array_key_exists('to', $headers) && $headers['to']) {
-        $recip_count += count(process_address_fld($headers['to']));
-    }
-    if (array_key_exists('cc', $headers) && $headers['cc']) {
-        $recip_count += count(process_address_fld($headers['cc']));
+    if ($reply_type == 'reply') {
```

Comment:
This is is unnecessary you just and other condition to **if (array_key_exists('cc', $headers) && $headers['cc'])** that tests also if reply it is reply_all